File: fullStackGregslistVue.client/src/models/Car.js
import { Profile } from "./Account.js"

export class Car {
  constructor(data) {
    this.id = data.id
    this.make = data.make
    this.model = data.model
    this.imgUrl = data.imgUrl
    this.body = data.body
    this.price = data.price
    this.description = data.description
    this.creatorId = data.creatorId
    this.seller = new Profile(data.seller)
  }
}

File: fullStackGregslistVue/Models/Car.cs
namespace fullStackGregslistVue.Models;

public class Car
{
  public int Id { get; set; }

  public string Make { get; set; }

  public string Model { get; set; }

  public string ImgUrl { get; set; }

  public string Body { get; set; }

  public int? Price { get; set; }

  public string Description { get; set; }

  public string CreatorId { get; set; }

  public Profile Seller { get; set; }
}
